db/patch70.sql

```diff
@@ -0,0 +1,32 @@
+#Theres already an unsed table relating to this, drop it
+DROP TABLE talk_link_types;
+#switch talks to innodb so FKs work
+ALTER TABLE `joindin`.`talks` ENGINE=INNODB;
+
+CREATE TABLE `talk_link_types` (
+  `ID` INT(11) NOT NULL AUTO_INCREMENT,
+  `display_name` VARCHAR(255) DEFAULT NULL,
+  PRIMARY KEY (`ID`)
+) ENGINE=INNODB;
+
+
+CREATE TABLE `joindin`.`talk_links`(
+  `ID` INT NOT NULL AUTO_INCREMENT,
+  `talk_id` INT NOT NULL,
+  `talk_type` INT NOT NULL,
+  `url` TEXT NOT NULL,
+  `timestamp` TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+  PRIMARY KEY (`ID`),
+  FOREIGN KEY (`talk_id`) REFERENCES `joindin`.`talks`(`ID`),
+  FOREIGN KEY (`talk_type`) REFERENCES `joindin`.`talk_link_types`(`ID`)
+);
+
+
+INSERT INTO `talk_link_types`(`ID`,`type`,`display_name`) VALUES
```

Comment:
@exussum12 - looks like we a missing the `display_name` values below - patch won't apply :)